rootfs/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM alpine:3.3
+
+RUN apk -U add ca-certificates && \
+    rm -Rf /var/cache/apk/*
+EXPOSE 8081
+ENV PORT 8081
+ENV HOST 0.0.0.0
+COPY usr/bin/monocular /usr/local/bin/monocular
```

Comment:
I am wondering if instead of assuming that the binary is in the host, make this dockerfile to build the binary from the source code.